.github/workflows/Ci.yml

```diff
@@ -0,0 +1,109 @@
+name: "CI: Run tests"
+
+# GitHub Actions triggers based on the PR's and each commits on develop and master
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [develop, master,feature/add_gha]

```

Comment:
Was the `feature/add_gha` branch added here to enable the CI workflow to run on this PR? We will need to remove this before merging, so a different solution will be needed to ensure these steps run for all PRs.

Reply:
Removed feature branch and workflow dispatch as we only build on PR's, develop and master